CHANGELOG.md

```diff
@@ -18,13 +18,15 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## [Unreleased]
 
 ### Fixes
-- Fix theme switcher not applying the new theme when the component's page is changed @mnajdova ([#911](https://github.com/stardust-ui/react/pull/911))
```

Comment:
No public change with this PR. See comment: https://github.com/stardust-ui/react/pull/911/files#r257390850